code/datums/uplink_items/uplink_general.dm

```diff
@@ -803,6 +803,22 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/bio_chip_implanter/stealth
 	cost = 45
 
+// CYBERNETICS
+
+/datum/uplink_item/cyber_implants
+	category = "Cybernetic Implants"
+
+/datum/uplink_item/cyber_implants/hackerman_deck
+	name = "Binyat Wireless Hacking System Autoimplanter"
+	desc = "A rare-to-find neural chip that allows its user to interface with nearby machinery \
```

Comment:
I do feel like it might be worth noting here (in perhaps vague, but still clear terms) that it emags stuff at range.

Reply:
done